hexa/user_management/utils.py

```diff
@@ -16,9 +16,5 @@ def default_device(user, confirmed=True):
             return device
 
 
-def is_two_factor_available(user):
-    return user.is_authenticated and user.has_feature_flag("two_factor")
```

Comment:
Since we remove the two_factor check, I thougth it would be better to merge the function with `has_configured_two_factor`